components/Picture.tsx

```diff
@@ -36,4 +40,7 @@ const StyledWrapper = styled.div`
 
 const StyledImage = styled(CldImage)`
   border-radius: 0.2em;
+  object-fit: contain;
+  width: ${(props) => (props.big === "true" ? "150px" : "125px")};
+  height: ${(props) => (props.big === "true" ? "150px" : "125px")};
```

Comment:
ich verstehs leider nicht ganz, oben hast du height und width in 300px geändert und mit dem Attribut "big" überschreibst du es aber dann?

Reply:
Nicht ganz: Width und Height oben beziehen sich auf die Auflösung des Bildes, d.h. das Bild hat 300x300px. Die Werte hier beziehen sich auf den Wrapper um das Bild herum. D.h. ich kann damit die Anzeigegröße (big oder default) bestimmen - die Auflösung bleibt aber gleich.

Reply:
💡